tests/run.py

```diff
@@ -527,6 +527,8 @@ def main():
                 cleanup_and_exit(tmpdir, -1);
         if('query_params' in test_params_dict):
             for query_param_dict in test_params_dict['query_params']:
+                if(test_name.find('java_genomicsdb_importer_from_vcfs') != -1):
+                    test_name = "1"
```

Comment:
?

Reply:
This is for replacing array name (which in run.py test_name is used for this end) with chromosome name... If you remember, we decided to remove array name from the import command line since the idea is that each TileDB array folder represents a chromosome interval.